### PR TITLE
SEC-1883: Upgrade Netty version to 4.1.61.Final

### PR DESCRIPTION
**Pull request by xjin-Confluent**
This will fix the following three CVEs brought by io.netty:netty-codec-http2 and io.netty:netty-codec-http.
https://confluentinc.atlassian.net/browse/SEC-1883
https://confluentinc.atlassian.net/browse/SEC-2016
https://confluentinc.atlassian.net/browse/SEC-2022

### Committer Checklist (excluded from commit message)
- [x] Verify design and implementation 
- [x] Verify test coverage and CI build status
- [x] Verify documentation (including upgrade notes)

### CODE REVIEW

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -103,7 +103,7 @@ versions += [
   mavenArtifact: "3.6.3",
   metrics: "2.2.0",
   mockito: "3.2.4",
-  netty: "4.1.50.Final",
+  netty: "4.1.61.Final",
```
Comment (omkreddy) [inline gradle/dependencies.gradle:106]: Can we use latest `4.1.62.Final` release?

**Comment by omkreddy** [comment]
nit: We use JIRA ID or "CONFLUENT:" in the PR title.

File: gradle/dependencies.gradle (hunk 1 above)
Comment (xjin-Confluent) [reply gradle/dependencies.gradle:106]: Make sense.

**Comment by xjin-Confluent** [comment]
> nit: We use JIRA ID or "CONFLUENT:" in the PR title.

SEC-1883 is a Jira ID. Do you mean put the text "Jira ID" in the title?

**Review by omkreddy — APPROVED** [review]
@xjin-Confluent Thanks for the PR. LGTM. We need to cherry-pick the commit to upto 2.7 branch.  We have updated the version in AK 2.8, so it will get synced to ccs 2.8

**Comment by omkreddy** [comment]
> > nit: We use JIRA ID or "CONFLUENT:" in the PR title.
> 
> SEC-1883 is a Jira ID. Do you mean put the text "Jira ID" in the title?
No. I have updated the PR title (ex: "SEC-1883: Upgrade Netty version to 4.1.61.Final"). We can follow similar pattern

**Comment by xjin-Confluent** [comment]
Cherry-pick results seems to carry multiple lines. Here is a simpler way: https://github.com/confluentinc/kafka/pull/539

**Comment by xjin-Confluent** [comment]
Forgot 2.6 merge: https://github.com/confluentinc/kafka/pull/540

**Pull request merged**